rudder-core/src/test/scala/com/normation/rudder/services/policies/write/WriteSystemTechniquesTest.scala

```diff
@@ -97,23 +107,31 @@ import net.liftweb.common.Loggable
 @RunWith(classOf[JUnitRunner])
 class WriteSystemTechniqueSpec extends Specification with Loggable with ContentMatchers with AfterAll {
 
+  //just a little sugar to stop hurting my eyes with new File(blablab, plop)
+  implicit class PathString(root: String) {
+    def /(child: String) = new File(root + "/" + child)
```

Comment:
still, there is no reasons to worsen the situation by using string with hardcoded separator